programmers/가운데-글자-가져오기.js

```diff
@@ -0,0 +1,8 @@
+function solution(s) {
+    
+    const even = s.length / 2;
+    const odd = (s.length / 2) - 0.5;
+    
+    return s.length % 2 === 1 ? s[odd] : s[even-1] + s[even]
```

Comment:
나눈 결과가 ```1```이면 어차피 ```true```이기에 조건식이 없어도 됩니다 :D
```suggestion
    return s.length % 2 ? s[odd] : s[even-1] + s[even]
```

Reply:
기억하겠습니다 ~! 감사합니다 !